app/config/config.yml

```diff
@@ -35,6 +35,7 @@ twig:
     globals:
         role_voir_confidentiel: "%role_voir_confidentiel%"
         authorizedStorageSize: "%junior_authorizedstoragesize%"
+        technical_to: "%technical_to%"
```

Comment:
I think a global variable like that is fine. It's only for twig so that's ok. An alternative would be to inject the container or the parameter bag as a global variable, but IMO this is less ideal as you expose way more than you should. I'm not aware of any other easy way to do this